include/rocshmem/rocshmem.hpp

```diff
@@ -597,7 +619,41 @@ __device__ ATTR_NO_INLINE void rocshmem_wave_sync_all();
  */
 __device__ ATTR_NO_INLINE void rocshmem_ctx_wg_sync_all(rocshmem_ctx_t ctx);
 
-__device__ ATTR_NO_INLINE void rocshmem_wg_sync_all();
+/**
+ * @brief registers the arrival of a PE at a barrier.
+ * The caller is blocked until the synchronization is resolved.
+ *
+ * In contrast with the shmem_barrier_all routine, shmem_team_sync only ensures
+ * completion and visibility of previously issued memory stores and does not
+ * ensure completion of remote memory updates issued via OpenSHMEM routines.
+ *
+ * This function must be invoked by a single thread within the PE.
+ *
+ * @param[in] handle GPU side handle.
+ * @param[in] team  Handle of the team being synchronized
+ *
+ * @return void
+ */
+__device__ ATTR_NO_INLINE void rocshmem_ctx_team_sync(
```

Comment:
The oshmem version is named `shmem_team_sync` and doesn't take the context as parameter. 

In general, when a procedure takes a context as input it doesn't need the team, because every shmem_ctx_t context is associated with a team when created.